test_output.csv

```diff
@@ -0,0 +1,1780 @@
+False,Not classified,,2024-04-02 02:00:00,losca_city_planning/202404020000/x/historic_preservation_overlay_zone,"[{'href': 'http://planning.lacity.gov/dcpapi/meetings/document/76410', 'title': '76410'}]","{'address': '3504 S. 5th Ave', 'name': 'Jefferson Park'}",https://planning.lacity.gov/dcpapi/meetings/api/all/boards/2024,2024-04-02 00:00:00,passed,,Historic Preservation Overlay Zone
+False,Not classified,meeting canceled,2024-04-02 02:00:00,losca_city_planning/202404020000/x/historic_preservation_overlay_zone,"[{'href': 'http://planning.lacity.gov/dcpapi/meetings/document/76411', 'title': '76411'}]","{'address': 'N/A', 'name': 'Banning Park, Vinegar Hill'}",https://planning.lacity.gov/dcpapi/meetings/api/all/boards/2024,2024-04-02 00:00:00,cancelled,,Historic Preservation Overlay Zone
+False,Board,Meeting Cancelled.,2024-04-01 02:00:00,losca_city_planning/202404010000/x/design_review_board,"[{'href': 'http://planning.lacity.gov/dcpapi/meetings/document/76363', 'title': '76363'}]","{'address': 'Meeting Cancelled', 'name': 'Colorado Boulevard'}",https://planning.lacity.gov/dcpapi/meetings/api/all/boards/2024,2024-04-01 00:00:00,cancelled,,Design Review Board
+False,Not classified,Canceled,2024-03-28 02:00:00,losca_city_planning/202403280000/x/historic_preservation_overlay_zone,"[{'href': 'http://planning.lacity.gov/dcpapi/meetings/document/76370', 'title': '76370'}]","{'address': 'Canceled due to lack of quorum.', 'name': 'Highland Park'}",https://planning.lacity.gov/dcpapi/meetings/api/all/boards/2024,2024-03-28 00:00:00,cancelled,,Historic Preservation Overlay Zone
+False,Not classified,Canceled,2024-03-28 02:00:00,losca_city_planning/202403280000/x/historic_preservation_overlay_zone,"[{'href': 'http://planning.lacity.gov/dcpapi/meetings/document/76375', 'title': '76375'}]","{'address': 'Canceled due to lack of agenda items.', 'name': 'Adams Normandie'}",https://planning.lacity.gov/dcpapi/meetings/api/all/boards/2024,2024-03-28 00:00:00,cancelled,,Historic Preservation Overlay Zone
+False,Not classified,Updated Agenda ,2024-03-28 02:00:00,losca_city_planning/202403280000/x/historic_preservation_overlay_zone,"[{'href': 'http://planning.lacity.gov/dcpapi/meetings/document/76378', 'title': '76378'}]","{'address': '1635 N Genesee Ave| 1334 N Genesee', 'name': 'Sunset Sq, Spaulding Sq'}",https://planning.lacity.gov/dcpapi/meetings/api/all/boards/2024,2024-03-28 00:00:00,passed,,Historic Preservation Overlay Zone
+False,Board,Canceled,2024-03-27 02:00:00,losca_city_planning/202403270000/x/design_review_board,"[{'href': 'http://planning.lacity.gov/dcpapi/meetings/document/76336', 'title': '76336'}]","{'address': 'N/A', 'name': 'Pacific Palisades'}",https://planning.lacity.gov/dcpapi/meetings/api/all/boards/2024,2024-03-27 00:00:00,cancelled,,Design Review Board
+False,Not classified,Canceled,2024-03-27 02:00:00,losca_city_planning/202403270000/x/h
```

Comment:
This file should not be committed into the repo. Like the filename implies, it should be just be used for testing the outpuot. We do not need a CSV file that is 1,780 lines long in order to run this spider or the tests. This file could be removed and nothing bad would happen. This file is not essential.